### PR TITLE
Add command line option to skip messages based on type

### PR DESCRIPTION
**Pull request by nhi-nguyen**

### CODE REVIEW

**Comment by nhi-nguyen** [comment]
Sample test results for `--skip-messages=page,alias,screen,Group,track `:
```
=== SKIP advance-group.json
=== SKIP group.json
=== RUN identify.json
--- PASS: identify.json (5.914051 seconds)
=== RUN advance-identify.json
--- PASS: advance-identify.json (3.548329 seconds)
=== SKIP page.json
=== SKIP screen.json
=== SKIP advance-screen.json
=== SKIP advance-track.json
=== SKIP track.json
=== SKIP advance-alias.json
=== SKIP alias.json
```

**Review by f2prateek — APPROVED** [review]

File: tester.go [hunk 1]
```diff
@@ -97,6 +110,13 @@ func (t *T) Test(invoker Invoker) error {
 
 			msgType := msg["type"].(string)
 
+			if t.SkipMessageType(msgType) {
+				events.Log("Skip message for fixture %{fixture}v", fixture)
```
Comment (f2prateek) [inline tester.go:114]: Is there a way to print skipped tests in the report as well?

Comment (nhi-nguyen) [reply tester.go:114]: yes, it's already been done. SEe the sample result in the comment above.

Comment (f2prateek) [reply tester.go:114]: ah perfect!

**Pull request merged**